### ISSUE TITLE
FR001: Single FaultRule should not require condition

### ISSUE THREAD

**Issue by adamhibbert** [apigee/apigeelint#146]
If FaultRules only contains a single fault rule. The condition requirement should not be mandatory.

In my case I have a single fault rule which delegates to a shared flow and fault conditions are dealt with inside the shared flow. The only workaround I have is to add a condition 1=1 or similar to this fault rule.

**Comment by DinoChiesa** [comment]
Could you avoid it also, by just using a DefaultFaultRule?  
And then you'd exploit the feature in Apigee that is designed for the purpose of handling all faults?

**Comment by kurtkanaskie** [comment]
Yes, you could put all your fault handling into the DefaultFaultRule as a single shared flow callout.
`
    <DefaultFaultRule name="ProxyDefaultFaultRule">
        <Step>
            <Name>FC-ProxyErrorHandling</Name>
        </Step>
        <AlwaysEnforce>true</AlwaysEnforce>
    </DefaultFaultRule>
    <FaultRules/>
`

**Comment by DinoChiesa** [comment]
The suggested change has now been implemented via PR #316
